services/QuillLMS/client/app/bundles/Teacher/containers/dashboard.tsx

```diff
@@ -22,14 +23,20 @@ import { GRAY_ARTICLE_FOOTER_BACKGROUND_COLOR, TEACHER_DASHBOARD_FEATURED_BLOG_P
 
 const MAX_VIEW_WIDTH_FOR_MOBILE = 1103
 
-const Dashboard = ({ onboardingChecklist, firstName, mustSeeWelcomeModal, mustSeeTeacherInfoModal, linkedToClever, featuredBlogPosts, showEvidencePromotionCard, subjectAreas, }) => {
+const MAY = 4
+
+const Dashboard = ({ onboardingChecklist, firstName, mustSeeWelcomeModal, mustSeeTeacherInfoModal, linkedToClever, featuredBlogPosts, showEvidencePromotionCard, subjectAreas, userId, classrooms, inTestEnv, }) => {
```

Comment:
Is `inTestEnv` leftover from earlier code?  I don't see it referenced anywhere else or being passed in.